### ISSUE TITLE
Add test images for each model, if possible

### ISSUE THREAD

**Issue by mathieuboudreau** [axondeepseg/nn-axondeepseg#3]
In order to complete the installation instructions. Also optional: show the resulting output that users should expect with these images in the README installation instructions

**Comment by hermancollin** [comment]
Yes this makes sense. I'll have to ask permission to use ROIs from VCU/UoT images to make them public. 
However, maybe it would make more sense to have these images in the model repos, and the `download_models` script could just download them as well for a test run.
